src/btree/bt_read.c

```diff
@@ -166,17 +238,18 @@ __las_page_instantiate(WT_SESSION_IMPL *session, WT_REF *ref)
     } * las_preparep;
     WT_BIRTHMARK_DETAILS *birthmarkp;
     WT_CACHE *cache;
-    WT_CURSOR *cursor;
+    WT_CURSOR *las_cursor;
     WT_CURSOR_BTREE cbt;
     WT_DECL_ITEM(las_prepares);
     WT_DECL_RET;
-    WT_ITEM las_key, las_value, next_las_key;
+    WT_ITEM las_key, _las_key, las_value, next_las_key;
+    WT_MODIFY_VECTOR modifies;
     WT_PAGE_LOOKASIDE *page_las;
     WT_PAGE *page;
-    WT_UPDATE *list[WT_MODIFY_ARRAY_SIZE], **listp, *upd;
-    wt_timestamp_t durable_timestamp, las_timestamp;
-    size_t allocated_bytes, notused, size, total_incr;
-    uint64_t birthmark_cnt, las_txnid, recno;
+    WT_UPDATE *mod_upd, *upd;
+    wt_timestamp_t durable_timestamp, _durable_timestamp, las_timestamp, _las_timestamp;
+    size_t notused, size, total_incr;
+    uint64_t birthmark_cnt, las_txnid, _las_txnid, recno;
```

Comment:
I really don't like the underscore prefixes here. Generally, we use underscores to flag things that are private to a namespace (because C forces that on us), and for structures that we'll never reference by that name:
```
WT_ITEM _foo, *foo;
foo = &_foo;    /* We never use _foo itself. */
```
but I don't see how either of those makes sense here? I'd strongly prefer something like `las_txnid_tmp`, or `las_txnid_notused`, or something, but underscore prefix doesn't tell me anything useful.

(To be fair, I was one of the people that started the whole `WT_ITEM _foo, *foo;` stuff in the code base, and I regret it now. )

Reply:
eff57c7

I've given everything a `_tmp` suffix. I also changed `__wt_find_lookaside_upd` which is where I started getting into this habit.